submissions/lalalilulu/hooli-style_popup/styles.css

```diff
@@ -30,10 +30,30 @@ nav {
     object-fit: cover;
 }
 
+.main-popup {
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way.

Reply:
thank you - the video helps a lot